src/@sdk/react/mutations.ts

```diff
@@ -17,6 +17,16 @@ export const useSignIn = (
     update: (proxy, data) => {
       if (data.data.tokenCreate.token) {
         setAuthToken(data.data.tokenCreate.token);
+        if (navigator.credentials && navigator.credentials.store) {
```

Comment:
shouldn't this be in one if condition?

Reply:
first condition can be put in maybe: `if(maybe(() => navigator.credentials.store))`